### PR TITLE
Add domovoi to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
- Ignore the Windows failure. It depends on `chalice` which is not available on Windows yet.

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by sodre** [comment]
@conda-forge/help-python the domovoi recipe is ready for review

File: recipes/domovoi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: c9d8d7f670bedaac86514a0e6e479be6cb120e70c8497f9eb3cfab587bc72b79
+  - url: https://raw.githubusercontent.com/kislyuk/{{ name }}/v{{ version }}/LICENSE
+    sha256: a8ad31b1c3f40dca5a84119351b8fa8ddc868edd77fad8a8ebf6d8f2d16fa4ae
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win or py27]
```
Comment (beckermr) [inline recipes/domovoi/meta.yaml]: Can this one be noarch: python?

**Comment by sodre** [comment]
I tried noarch first? It is failing on Windows because of the chalice dependency. 

It also includes a ‘script’ which does not work with conda noarch.  

I’ll open a ticket upstream to replace the script if there is interest from conda community.

**Comment by beckermr** [comment]
The windows failure for noarch is not a big deal. The script won’t work of course.

**Comment by sodre** [comment]
Thanks!

**Pull request merged**